src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -304,10 +304,24 @@ function prepareAtomic(a) {
     size: a.$.size,
     description: a.$.description,
     isDiscrete: a.$.discrete == 'true',
+    isComposite: a.$.composite == 'true',
     isSigned: a.$.signed == 'true',
-    isString: a.$.string == 'true',
-    isLong: a.$.long == 'true',
-    isChar: a.$.char == 'true'
+    isString: a.$.string == 'true' || a.$.name.toLowerCase().includes('string'),
```

Comment:
This seems over-broad.  Should restrict it to the actual string types?

Reply:
Fixed however.